### PR TITLE
Tooltips )

### PR DESCRIPTION
**Pull request by MatveiNikita**
изображение

It was very intresting to create this page

### CODE REVIEW

File: Homework№6/cssCode/index.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.tooltip {
+    margin: 10px;
+    border: 2px solid pink;
+    position: relative;
+    display: inline-block;
+}
+
+.tooltip_content {
```
Comment (IvanCeban) [inline Homework№6/cssCode/index.css:19]: Great css optimization!

File: Homework№6/cssCode/index.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    /* Position the tooltip */
+    position: absolute;
+    z-index: 1;
+}
+
+.tooltip:hover .tooltip_content {
+    visibility: visible;
+}
+
+.tooltip[data-placement="top"] .tooltip_content {
```
Comment (IvanCeban) [inline Homework№6/cssCode/index.css:37]: using data-placement - nice idea!

**Review by IvanCeban — APPROVED** [review]
Great job!
Just missing the triangles (arrows of  tooltips). Would be nice to add them as well

File: Homework№6/cssCode/index.css (hunk 1 above)
Comment (MatveiNikita) [reply Homework№6/cssCode/index.css:19]: I don't have ideas how can I do this
But, I'll try to do it

**Pull request merged**